### PR TITLE
Update Game Over Message

### PR DESCRIPTION
**Pull request by RidhikaJoshi**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by Harshit-3905 — APPROVED** [review]

**Review by RidhikaJoshi — COMMENTED** [review]
Reviewed and Updated Changes

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
```
Comment (RidhikaJoshi) [inline index.html:50]: <h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>

Comment (RidhikaJoshi) [inline index.html:50]: <h2>Game over yeahh!!!</h2>

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (RidhikaJoshi) [inline index.html]: <h2>Game over yeahh!!!</h2>
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (RidhikaJoshi) [inline index.html]: <h2>Game over yeahh!!!</h2>

```suggestion
<h2>Game over yeahh!!!</h2>
```

**Pull request merged**